src/main/java/uk/gov/justice/hmpps/casenotes/model/OffenderCaseNote.java

```diff
@@ -78,7 +82,7 @@
 
     @Builder.Default
     @SortComparator(AmendmentComparator.class)
-    @OneToMany(cascade = CascadeType.ALL, mappedBy = "caseNote")
+    @OneToMany(cascade = {PERSIST, MERGE, REFRESH, DETACH}, mappedBy = "caseNote")
```

Comment:
would be good to have a single line comment in the code explaining why not doing `ALL`

Reply:
otherwise someone will add back in.  Is there a test for this - i.e. if I change it to `ALL` will a test then break :pray: